kitsune/coolsearch/static/coolsearch/js/actions/SearchActionCreator.js

```diff
@@ -0,0 +1,23 @@
+/* globals $:false */
+
+import ResultsActionCreator from './ResultsActionCreator.js';
+
+
+var SearchActionCreator = {
+  runSearch: function (dataType, payload) {
+    var apiUrl = '/api/2/coolsearch/search/' + dataType + '/';
```

Comment:
ES6 syntax tip: This could be written as ``var apiUrl = `/api/2/coolsearch/search/${dataType}/`;``.